app/src/main/java/com/joocoding/android/app/githubsearch/MainAdapter.kt

```diff
@@ -0,0 +1,44 @@
+package com.joocoding.android.app.githubsearch
+
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.joocoding.android.app.githubsearch.databinding.ItemRepositoryBinding
+import com.joocoding.android.app.githubsearch.model.response.RepositoryResponse
+
+class MainAdapter() : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

Comment:
현재는 ViewHolder가 하나여서 RecyclerView.ViewHolder로 하지 않아도 될 것 같아요!
자연스럽게 onBind에서 is 키워드로 검증하는 로직이 빠질 수 있겠어요

```suggestion
class MainAdapter() : RecyclerView.Adapter< MainViewHolder>() {
```